### PR TITLE
Upgrade ansible to 2.9.14

### PR DESCRIPTION
**Pull request by OlleLarsson**
**What this PR does / why we need it**:
Upgrades ansible version.

**Which issue this PR fixes**:
NA

**Special notes for reviewer**:

**Checklist:**

- [ ] Added relevant notes to [WIP-CHANGELOG.md](https://github.com/elastisys/ck8s-cluster/blob/master/WIP-CHANGELOG.md)
- [ ] Proper commit message prefix on all commits
- Is this changeset backwards compatible for existing clusters? Applying:
    - [ ] is completely transparent, will not impact the workload in any way.
    - [ ] requires running a migration script.
    - [ ] will create noticeable cluster degradation.
          E.g. logs or metrics are not being collected or Kubernetes API server
          will not be responding while upgrading.
    - [ ] requires draining and/or replacing nodes.
    - [ ] will change any APIs.
          E.g. removes or changes any CK8S config options or Kubernetes APIs.
    - [ ] will break the cluster.
          I.e. full cluster migration is required.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,12 +20,28 @@ Currently we support three cloud providers: Exoscale, Safespring, an CityCloud.
...
+sudo apt update
+sudo apt install software-properties-common
+sudo add-apt-repository --update ppa:ansible/ansible-2.9
+sudo apt install ansible
+
+# On Ubuntu 20.04 - 2.9.6+dfsg-1 is directly available in packages
+sudo apt update
+sudo apt install ansible
+```
+
```
Comment (simonklb) [inline README.md]: I don't really feel this is the place to put package installation instructions. If you don't know how to install the dependencies we do link the various web resources in the requirements list above.

**Review by simonklb — APPROVED** [review]
LGTM with the removal from the readme (ideally I would like to be become shorter and more to the point than it is right now as well).

If we really want a more extensive documentation in this repository I would be more fine with putting it under `docs/` or maybe INSTALL.md.

Also, I'm currently working on making the Docker image more user friendly, hopefully that should make package management less of an issue for users eliminate the need for cluttering your base system. :partying_face:

File: README.md (hunk 1 above)
Comment (OlleLarsson) [reply README.md]: Done, I've removed this section.
I mean the installation instruction is pretty visible on the ansible docs page so i'd imagine that most people shouldn't have any issues installing :)

**Review by danielharr — APPROVED** [review]
LGTM!

**Pull request merged**